### DISCUSSION TITLE
Might it be helpful to bundle ./configure with the pgbouncer tarball? Or in the release tags?

### DISCUSSION THREAD

**Discussion by lembark** [pgbouncer/pgbouncer#818] [General]
Same result from
  gzip -dc < ~-/pgbouncer-1.18.0.tar.gz | cpio -idv;
or 
  git clone https://github.com/pgbouncer/pgbouncer.git;
  cd pgbouncer;
  git checkout pgbouncer_1_18_0;

less README.md
shows:
  When dependencies are installed just run:
  
      $ ./configure --prefix=/usr/local
      $ make
      $ make install

$ ./configure --prefix=/opt/pgbouncer;
bash: ./configure: No such file or directory

$ find . -name 'configure';
$

$ make configure;
Makefile:134: /var/tmp/Build/pgbouncer/lib/mk/antimake.mk: No such file or directory
make: *** No rule to make target '/var/tmp/Build/pgbouncer/lib/mk/antimake.mk'.  Stop.

I can understand perhaps having to build the configure working in with the checked-out tag, but at least the Makefile might want to inlcude "configure" as a target; I'm not sure I understand the reason for leaving ./configure out of the release tarball. 

Q: What steps am I missing in the README.md (there isn't a separate INSTALL document)?

Thanks

**Comment by eulerto** [comment]
I don't know how did you download `pgbouncer-1.18.0.tar.gz`. If you download it from the [releases](https://github.com/pgbouncer/pgbouncer/releases) page, all tarball files certainly contain the `configure` file. If you are using git to obtain the source code, you need to generate the `configure` file using `autogen.sh` as stated in the [Building from Git](https://github.com/pgbouncer/pgbouncer#building-from-git) section.
